modules/rmarkdown/main.nf

```diff
@@ -0,0 +1,92 @@
+// Import generic module functions
+include { initOptions; saveFiles; getSoftwareName } from './functions'
+include { dump_params_yml } from "./parametrize"
+
+params.options = [:]
+options        = initOptions(params.options)
+params.parametrize = true
+params.implicit_params = true
+params.meta_params = true
+
+process RMARKDOWN {
```

Comment:
```suggestion
process RMARKDOWNNOTEBOOK {
```

RMarkdown is a separate thing in principle (could have a module rendering a plain Rmarkdown document to a HTML), and would be more consistent with JUPYTERNOTEBOOK

Reply:
Could even think about moving the modules to `custom`?